assemblies/dynamic-plugins/assembly-third-party-plugins-installation.adoc

```diff
@@ -0,0 +1,114 @@
+[id="assembly-third-party-plugins-installation"]
+= Third-party plugins in {product}
+:context: assembly-third-party-plugins-installation
+
+In {product}, third-party dynamic plugins can be integrated seamlessly, enabling the use of external plugins alongside native ones. This flexibility allows you to enhance the platform functionality without modifying its core structure. To add third-party dynamic plugins, you can either export them as standalone packages or wrap them in custom wrappers, especially if you want to adjust the dependencies or configurations of a plugin for compatibility.
+
+The third-party plugins are typically packaged as dynamic plugins, which means they can be loaded at runtime, enabling you to easily manage external modules without rebuilding the entire application. The process involves creating a wrapper that exports the plugin code, ensuring that dependencies are correctly bundled or marked as shared, depending on their relationship to the {product-short} environment.
+
+To integrate a third-party plugin into {product-short}:
+
+. First, obtain the plugin's source code.
+. Export the plugin as a dynamic plugin package. See xref:proc-export-third-party-plugins-rhdh_{context}[].
+. Package and publish the dynamic plugin. See xref:proc-package-third-party-dynamic-plugin_{context}[].
+. Install the plugin in the {product-short} environment. See xref:proc-install-third-party-plugins-rhdh_{context}[].
+
+The third-party plugins include:
+
+Backend plugins::
++
+--
+To ensure compatibility with the dynamic plugin support and enable their use as dynamic plugins, existing backend plugins must be compatible with, the new backend system. Additionally, these plugins must be rebuilt using a dedicated CLI command.
+
+The new backend system is created using `createBackendPlugin()` or `createBackendModule()`. You must export the new backend system as the default export from either the main package or an alpha package (if support exists in alpha). The dynamic export mechanism automatically identifies private dependencies, and adds them to the `bundleDependencies` field in the `package.json` file. This mechanism ensures that the dynamic plugin package is self-contained, with private dependencies bundled within a private `node_modules` folder.
+
+The backend plugins contains two types of dependencies, including:
+
+* *Shared dependencies* are provided by an application and listed as `peerDependencies` in `package.json` file, not bundled in the dynamic plugin package. For example, by default, all `@backstage` packages are shared.
```

Comment:
instead of "an application" we could probably instead refer to the main RHDH application here.  The term "an application" is better used when this functionality is made portable and installable in any backstage, but for probably for the product docs referring to the RHDH application here is better.